common/src/main/java/org/geysermc/floodgate/util/DeviceOs.java

```diff
@@ -39,15 +39,16 @@ public enum DeviceOs {
     OSX("macOS"),
     AMAZON("Amazon"),
     GEARVR("Gear VR"),
-    HOLOLENS("Hololens"),
+    HOLOLENS("Hololens"), // Removed from protocol but kept to keep numbers the same
     UWP("Windows"),
     WIN32("Windows x86"),
     DEDICATED("Dedicated"),
     TVOS("Apple TV"),
-    PS4("PS4"),
+    SONY("PlayStation"),
```

Comment:
Just add a Javadoc indicating this isn't just for PS4. We can change the string though.